src/app/components/client/ExposureCard.tsx

```diff
@@ -18,101 +18,91 @@ import {
+  exposureType: ExposureType;
   exposureDetailsLink: string;
   dateFound: string;
-  statusPillType: string;
-  statusPillContent: string;
+  statusPillType: StatusPillType;
 };
 
 type DetailsFoundProps = {
-  whichExposed: string; // family | email | phone | address
+  whichExposed: string; // family | email | phone | address | credit card | password
   num: number;
   icon: ReactElement;
 };
 
-export const ExposureCard = (props: Props) => {
+export const ExposureCard = (props: ExposureCardProps) => {
   const {
     exposureImg,
     exposureName,
     exposureType,
     exposureDetailsLink,
-    dateFound,
-    statusPillContent,
     statusPillType,
   } = props;
 
+  const l10n = useL10n();
   const [detailsOpen, setDetailsOpen] = useState(false);
 
   const DetailsFoundItem = (props: DetailsFoundProps) => {
     let headline, description;
     if (props.whichExposed === "family") {
-      headline = "Family members";
-
-      description = `We found ${props.num} family member`;
-      if (props.num > 1) {
-        description = `We found ${props.num} family members`;
-      }
+      headline = l10n.getString("exposure-card-family-members");
+      description = l10n.getString("exposure-card-num-found", {
+        exposure_num: props.num
+      });
     }
-
     if (props.whichExposed === "email") {
-      headline = "Email";
-
-      description = `We found ${props.num} email address`;
-      if (props.num > 1) {
-        description = `We found ${props.num} email addresses`;
-      }
+      headline = l10n.getString("exposure-card-email");
+      description = l10n.getString("exposure-card-num-found", {
+        exposure_num: props.num
+      });
     }
 
     if (props.whichExposed === "phone") {
-      headline = "Phone number";
-
-      description = `We found ${props.num} phone number`;
-      if (props.num > 1) {
-        description = `We found ${props.num} phone numbers`;
-      }
+      headline = l10n.getString("exposure-card-phone-number");
+      description = l10n.getString("exposure-card-num-found", {
+        exposure_num: props.num
+      });
     }
 
     if (props.whichExposed === "address") {
-      headline = "Address";
-
-      description = `We found ${props.num} address`;
-      if (props.num > 1) {
-        description = `We found ${props.num} addresses`;
-      }
+      headline = l10n.getString("exposure-card-address");
+      description = l10n.getString("exposure-card-num-found", {
+        exposure_num: props.num
+      });
     }
 
     return (
-      <>
+      <div className={styles.detailsFoundItem}>
         <dt>
           <span className={styles.exposureTypeIcon}>{props.icon}</span>
           {headline}
         </dt>
-        <dl>{description}</dl>
-      </>
+        <dd>{description}</dd>
+      </div>
     );
   };
   const elementCard = (
     <div>
       <div className={styles.exposureCard}>
         <div className={styles.exposureHeader}>
           <ul className={styles.exposureHeaderList}>
```

Comment:
I missed this the first time (annoying, I know...), but imagining this list being read out to someone, I don't think it would make much sense to them. Could you convert it into a `<dl>` with the `<dt>`s containing descriptions of every heading (e.g. "Exposure name", "Date found", etc. - we should be able to use the same labels that we'll use as titles above the cards), and then visually hide the `<dt>`s? See [here](https://github.com/mozilla/blurts-server/blob/6422ebf1af7a8529289a30201313c3af41ef8e78/src/app/components/client/Chart.module.scss#L52-L63) for an example of how to visually hide something while keeping it available to screen readers.

Reply:
I meant

```html
<dl>
  <dt class="visually-hidden">Breached Company</dt>
  <dd>Familytree.com</dd>
  <dt class="visually-hidden">Date found</dt>
  <dd>11/09/23</dd>
</dl>
```

rather than the current structure:

```html
<ul>
  <li>Familytree.com</li>
  <li>11/09/23</li>
</ul>
```